locales/cy/stop-screens.json

```diff
@@ -40,12 +40,12 @@
   "dob_mismatch_agent_can_correct_link_text": "To be translated: dob_mismatch_agent_can_correct_link_text",
   "empty_psc_list_body": "To be translated: empty_psc_list_body",
   "empty_psc_list_link": "To be translated: empty_psc_list_link",
-  "super_secure_main_title": "To be translated: super_secure_main_title",
-  "super_secure_statement_1": "To be translated: super_secure_statement_1",
-  "super_secure_statement_2": "To be translated: super_secure_statement_2",
-  "super_secure_statement_3_1": "To be translated: super_secure_statement_3_1",
-  "super_secure_statement_3_2": "To be translated: super_secure_statement_3_2",
-  "super_secure_statement_3_3": "To be translated: super_secure_statement_3_3",
+  "super_secure_main_title": "Ni allwch ddarparu manylion qwiriad ar gyfer PRhA y cwmni hwn ar-lein",
+  "super_secure_statement_1": "Mae manylion y cwmni hwn a’u pobl sydd â rheolaeth arwyddocaol (PRhA) wedi’u diogelu rhag cael eu dangos ar gofrestr Tŷ’r Cwmnïau.",
```

Comment:
I've been using unicode values for all these characters... is it not necessary for apostrophes then?
and there's a `ŷ` here - is that ok?